bosh-stemcell/spec/stemcells/ubuntu_trusty_spec.rb

```diff
@@ -177,6 +184,24 @@
     end
   end
 
+  context 'installed by bosh_softlayer_agent_settings', {
+      exclude_on_aws: true,
+      exclude_on_vcloud: true,
+      exclude_on_vsphere: true,
+      exclude_on_warden: true,
+      exclude_on_azure: true,
+      exclude_on_openstack: true,
+  } do
+    describe file('/var/vcap/bosh/agent.json') do
+      it { should be_valid_json_file }
+      it { should contain('"Type": "File"') }
+      it { should contain('"MetaDataPath": ""') }
+      it { should contain('"UserDataPath": "/var/vcap/bosh/user_data.json"') }
```

Comment:
@cppforlife Hi Dmitriy, I am so sorry,  you are right, and it is my mistake because the code in the branch for my PR is a little outdate, I just updated the PR with some new stages for softlayer stemcell builder, for example, install open-iscsi, multipath-tool, as you know, we need use open-iscsi and multipath-tools to handle the works related with persistent disk in softlayer